df.history.xml

```diff
@@ -200,8 +200,43 @@
             <stl-vector name="wanted">
                 <pointer>
                     <int32_t name='entity_id' ref-target='historical_entity'/>
-                    <stl-vector type-name='int32_t'/>
-                    <stl-vector type-name='int32_t'/>
+                    <stl-vector name='types'>
+                        <enum base-type='int32_t'>
+                            0
+                            <enum-item name='Hero'/>
```

Comment:
I have added all these to the pull request, taking their names from the `<rep_*>` elements of the XML dump.